amenity-points.mss

```diff
@@ -567,7 +567,12 @@
 
   [feature = 'shop_other'][zoom >= 17] {
     marker-fill: @shop-icon;
-    marker-width: 6;
+    [zoom < 18] {
```

Comment:
Why not try to place all generic dots here to avoid duplication for each feature? At least the filter has to be duplicated though.